src/font/install_windows.go

```diff
@@ -42,9 +42,14 @@ func install(font *Font, admin bool) error {
 	}
 
 	// Add registry entry
-	reg := registry.LOCAL_MACHINE
-	if !admin {
+	var reg registry.Key
```

Comment:
I would instantiate these as admin, then change the values when you're not admin. Assuming admin is the most common option. That way the if/else isn't necessary. I try to avoid if/else at all times.

Reply:
Thanks! I've never go'ed (gone?) before :-)